src/System.Windows.Forms.Design.Editors/src/System/ComponentModel/Design/ObjectSelectorEditor.cs

```diff
@@ -92,12 +92,12 @@ public static void ApplyTreeViewThemeStyles(TreeView treeView)
         }
         private static ComCtl32.TVS_EX TreeView_GetExtendedStyle(IntPtr handle)
         {
-            return (ComCtl32.TVS_EX)User32.SendMessageW(handle, (User32.WindowMessage)NativeMethods.TVM_GETEXTENDEDSTYLE, IntPtr.Zero, IntPtr.Zero);
+            return (ComCtl32.TVS_EX)User32.SendMessageW(handle, User32.WindowMessage.TVM_GETEXTENDEDSTYLE, IntPtr.Zero, IntPtr.Zero);
```

Comment:
Likewise, this has to be 
```suggestion
            return (ComCtl32.TVS_EX)User32.SendMessageW(handle, (User32.WindowMessage)TVM.GETEXTENDEDSTYLE);
```

The last two parameters are optional.